tests/other_test.go

```diff
@@ -0,0 +1,57 @@
+// Tests in this file are NOT run in the PR pipeline. They are run in the continuous testing pipeline along with the ones in pr_test.go
+package test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
+)
+
+func TestRunBasicExample(t *testing.T) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:      t,
+		TerraformDir: "examples/basic",
+		Prefix:       "basic",
+		Region:       "us-south",
+	})
+
+	output, err := options.RunTestConsistency()
+
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
+
+func TestRunHybridExample(t *testing.T) {
```

Comment:
As per https://github.com/terraform-ibm-modules/terraform-ibm-hpcs/pull/166#issuecomment-2006814770 we cannoy add a test for this example - please remove it

Reply:
@ocofaigh hybrid test removed